src/intervals.jl

```diff
@@ -0,0 +1,102 @@
+#= 
+This file contains code for calculating intervals for plotting purposes.
+These functions should, at minimum, take in some form of data input and
+return a tuple (min, max) of the limits corresponding to the function =#
```

Comment:
```suggestion
return a tuple (min, max) of the limits corresponding to the function 
=#
```
Just a minor style nitpick